app/views/layouts/_javascripts.html.erb

```diff
@@ -36,7 +36,7 @@
 <% end %>
 
 <script type="text/javascript">$j = jQuery.noConflict();</script>
-<%= javascript_include_tag "jquery.scrollTo.min", "jquery.livequery.min", "rails", "application", "bootstrap/bootstrap-dropdown.min", "jquery-shuffle", "jquery.tokeninput.min", "jquery.trap.min", "ao3modal.min", "jquery.cookie",  skip_pipeline: true %>
+<%= javascript_include_tag "jquery.scrollTo.min", "jquery.livequery.min", "rails", "application", "bootstrap/bootstrap-dropdown.min", "jquery-shuffle", "jquery.tokeninput.min", "jquery.trap.min", "ao3modal.min", "jquery.cookie", "jquery.timeago.min", skip_pipeline: true %>
```

Comment:
I would add the timeago library only to _user_history.html.erb instead of the common layout, like how we include select_all only on the wrangling UI pages.

Reply:
I think we should consider using it on all the times myself but I shall do as you say.